internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1283,6 +1283,94 @@ public void testBootstrap(DaVinciConfig daVinciConfig) throws Exception {
     }
   }
 
+  @Test(timeOut = TEST_TIMEOUT, dataProvider = "dv-client-config-provider", dataProviderClass = DataProviderUtils.class)
+  public void testBootstrapSubscription(DaVinciConfig clientConfig) throws Exception {
```

Comment:
Since `DA_VINCI_SUBSCRIBE_ON_DISK_PARTITIONS_AUTOMATICALLY` is already true by default, we should instead write a test where it's set to false.